WebApp/static/core/js/views/tabs/RightContainer.js

```diff
@@ -416,7 +416,7 @@ Ext.define('Ext.dirac.views.tabs.RightContainer', {
                   isLoaded : isLoaded,
                   bodyStyle : {
                     background : '#AAAAAA',
-                    backgroundImage : 'url(/DIRAC/static/core/img/wallpapers/dirac_background_6.png)',
+                    backgroundImage : 'url('+GLOBAL.BACKGROUND+')',
```

Comment:
This is mostly my ignorance with js, so can you explain why at this line you used `+GLOBAL.BACKGROUND+` and at line 83 only `GLOBAL.BACKGROUND` ?

Reply:
In the line 83 a setter method is used. The wallpaper is a widget...  The setter method is doing all required rendering, etc. 
+GLOBAL.BACKGROUND+ is used inside a custom CSS property (bodyStyle) as you can see in line 417.  I set the background using CSS element. You can see how you can set a background using CSS: https://www.w3schools.com/cssref/pr_background-image.asp